core/src/test/java/edu/ucr/cs/riple/core/ConfigurationTest.java

```diff
@@ -205,6 +206,41 @@ public void testConfigFilesHaveDifferentUUID() {
     }
   }
 
+  @Test
+  public void testAnalysisModeFlags() {
+    List<CLIFlag> baseFlags = new ArrayList<>(requiredFlagsCli);
+    Config config;
+    // Check mode downstream dependency off.
+    config = new Config(makeCommandLineArguments(requiredFlagsCli));
```

Comment:
Should this be taking `baseFlags` here, since we already defined it? Otherwise I'd expect the definition of `baseFlags` to happen below this line (and after the following assertEquals)

Reply:
Updated it 7569c5ed1b34905c7dcaaf96668fc7f24beed225